docs/xml/nginx/changes.xml

```diff
@@ -5,6 +5,73 @@
 <change_log title="nginx">
 
 
+<changes ver="1.27.2" date="2024-10-02">
+
+<change type="feature">
+<para lang="ru">
+SSL-сертификаты, секретные ключи и списки CRL теперь кешируются
+при загрузке на старте или во время переконфигурации.
+</para>
+<para lang="en">
+loading SSL certificates, secret keys, and CRLs is now cached
```

Comment:
I'd remove "loading": SSL certificates, ... are now cached ...

Reply:
This may look ambiguous because it is loading that is cached, but I may be wrong.